datasets/us/trade_csl/crawler.py

```diff
@@ -8,10 +8,40 @@
 from zavod import Context, Entity
 from zavod import helpers as h
 
+#     Programs falling under the 'is_ofac=TRUE' check:
```

Comment:
I see the `is_ofac` check is somewhere below, maybe move that documentation there?

Reply:
I just looked into what this actually means. IIUC, this means that we emit these entities with the `ofac-<entity_number>` ID so that they match without a merge? Let's add a sentence about this here to explain this speciality.